.github/workflows/goreleaser.yaml

```
@@ -24,7 +24,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: "^1.18"
+          go-version: "^1.20"
```

```suggestion
          go-version-file: go.mod
          cache: true
```
Can we use this config instead (similar to our other projects)?

---

+ we then probably want to increase the go version in go.mod to 1.20

---

Why should we force anyone else building openshift-routes to use Go 1.20 by setting `go 1.20` in go.mod? The go.mod `go 1.18` indicates a compatibility, and the earliest Go version that can build openshift-routes is 1.18, so I wouldn't set 1.20 here

---

Regarding `cache: true`, you are correct. Alternatively, we could use `actions/setup-go@v4` which enables caching by default

---

ok, thx; I approved the PR already.